centrifuge-app/src/pages/IssuerCreatePool/index.tsx

```diff
@@ -217,13 +217,18 @@ const CreatePoolForm: React.VFC = () => {
     setTimeout(() => setWaitingForStoredIssuer(false), 10000)
   }, [])
 
+  const txMessage = {
+    immediate: 'Create pool',
+    propose: 'Submit pool proposal',
+    notePreimage: 'Noting preimage (fast track)',
```

Comment:
This isn't really fast-tracking it yet, it's just what enables the council to then fast track it afterwards, so I think we should change the copy a bit:

```suggestion
    notePreimage: 'Noting preimage',
```